packs/data/journals.db/archetypes.json

```diff
@@ -2553,6 +2561,423 @@
actionspf2e.Shove]{Shove}, or @Compendium[pf2e.actionspf2e.Trip]{Trip} attempt.</p><hr /><p>Your study of another allows you to mirror their movements subtly, even in the heat of combat. You study the successful maneuver of an enemy and instinctively learn to do it. The next time you take the same action they did, you gain a +1 circumstance bonus to your Athletics check, or a +2 circumstance bonus if the enemy critically succeeded on their triggering action. You lose this benefit if you don't use it before the end of your next turn.</p><p></p><h2>@UUID[Compendium.pf2e.feats-srd.ossGwqEvC6ifPwgT]{Swap Reflections} Feat 8</h2><p><strong>Prerequisites </strong>Alter Ego Dedication</p><p><strong>Frequency</strong> once per 10 minutes</p><hr /><p>You've learned how to use even your reflection to your advantage. You send your own reflection to forcibly swap places with the reflection of an enemy, pulling each of you through opposite ends and switching places. The enemy must be within 120 feet, you must both be adjacent to reflective surfaces (such as glass, mirrors, or calm water), and you must be able to see the target's reflection and have line of effect to it. The target attempts a @Check[type:will|dc:resolve(@actor.system.attributes.classOrSpellDC.value)] save against your spell DC or class DC, whichever is higher.</p><hr /><p><strong>Critical Success</strong> The spell has no effect.</p><p><strong>Success</strong> You swap positions with your target or arrive adjacent to your target, whichever your target prefers.</p><p><strong>Failure</strong> You swap positions with your target or arrive adjacent to your target, whichever you prefer.</p><p><strong>Critical Failure</strong> You instantaneously swap positions with your target, and the target becomes trapped in its own reflection for 1 minute. Once on each of its turns, the target can spend 1 action to attempt another Will save to escape. If the save succeeds, the effect ends early.</p><p></p><h2>@UUID[Compendium.pf2e.feats-srd.kC92uxCvrxDkrCpO]{Borrow Memories} Feat 14</h2><p><strong>Prerequisites </strong>Alter Ego Dedication</p><hr /><p>Knowing how someone thinks makes it much easier to know where they'll be, or even replace them altogether. You can cast <em>@Compendium[pf2e.spells-srd.Mind Probe]{Mind Probe}</em> as an innate occult spell once per day, using your class DC or spell DC, whichever is higher.</p><p>If you cast <em>mind probe</em> on a target you're studying for the purpose of Assuming their Role, and the target fails their save, you can delay the spell from coming into effect until you Assume their Role. When you use the spell this way, you can Sustain it up to 10 times during its duration, asking a question each time. The spell lasts until you ask all 10 questions or are no longer Assuming the Role, whichever comes first. You don't need to Sustain the Spell to extend its duration-only when asking a question-nor do you need to be in range or have line of effect to Sustain it.</p>",
```

Comment:
We don't use `<br>` tags. Do line breaks with new paragraphs `<p>First paragraph.</p><p>Second paragraph.</p>`

Also looks like there's some blank paragraphs in here `</p><p></p><h2>` we can ditch those.